app/controllers/checkouts_controller.rb

```diff
@@ -47,13 +48,31 @@ def destroy
     end
   end
 
+  # DELETE /checkouts
+  def return_all
```

Comment:
I'm thinking we may want to create a different action that returns a checkout by adjusting the return time without destroying the checkout.  It could be POST update or a new POST route. Unless you think it makes more sense to have the DELETE route be the return action.  For now, I'm thinking it might be easiest to rename this to `destroy_all` and have another PR implement the soft delete/return functionality.  What do you think?

Reply:
I had misinterpreted the single destroy action as a temporary implementation for returns and followed that lead for this action, but since end goal functionality is that return history is retained and viewable we should change this action as you suggest and implement the soft return with a future PR.

Reply:
Although, on further thought, is it necessary to have a mass hard delete?

Reply:
You're right.  I don't think it makes sense to have `delete_all` without a good use case (which I can't think of one right now).

Reply:
Ok. Should we just leave it for now and remove it in the followup PR properly implementing returns, or should I scrap it before we merge?